circle.yml

```diff
@@ -0,0 +1,26 @@
+machine:
+  environment:
+    PACKAGE_ROOT: /home/ubuntu/cufflinks
+    PLOTLY_CONFIG_DIR: ${HOME}/.plotly
+    PLOTLY_PYTHON_VERSIONS: 2.7.8 3.3.3 3.4.1
```

Comment:
these are the versions that the plotly-python client tests against, so we'll test cufflinks against these same versions.